dcos_test_utils/marathon.py

```diff
@@ -111,7 +111,7 @@ def get_app_service_endpoints(self, app_id: str) -> typing.List[Endpoint]:
         r.raise_for_status()
         data = r.json()
         res = [Endpoint(t['host'], t['ports'][0], t['ipAddresses'][0]['ipAddress'])
-               if len(t['ports']) is not 0
+               if len(t['ports']) != 0
```

Comment:
This could probably just be

```suggestion
               if t.get('ports')
```

To guard against `ports` not existing, as well as `0` and `None` being `False`.

Reply:
True, but that would also change the semantics. This code also happens to test that such (absence of) values are not returned, so I'd avoid changing that without some reason.